confgenerator/testdata/invalid/linux/logging-receiver_syslog_type_invalid_parameter_listen_port_is_not_an_int/golden_error

```diff
@@ -1,2 +1 @@
-the agent config file is not valid YAML. detailed error: yaml: unmarshal errors:
-  line 6: cannot unmarshal !!str `abc` into uint16
\ No newline at end of file
+the agent config file is not valid. detailed error: cannot unmarshal string into Go struct field UnifiedConfig.Logging of type uint16
```

Comment:
This one looks a bit different from others as it does not print the config snippet. Is that expected?

Reply:
No, I don't know why it's missing the line number and snippet. Nor why it says `unmarshal string into [...] uint16`. Worth investigating later.